extensions/eo/examples/landsat8-item.json

```diff
@@ -41,14 +41,17 @@
             ]
         ]
     },
-    "collection": "landsat-8-l1",
     "properties": {
+        "platform": "landsat-8",
+        "instrument": ["oli", "tirs"],
         "datetime": "2018-10-01T01:08:32.033Z",
-        "eo:cloud_cover": 78,
         "view:sun_azimuth": 168.8989761,
         "view:sun_elevation": 26.32596431,
+        "view:off_nadir": 0,
         "landsat:path": 107,
-        "landsat:row": 18
+        "landsat:row": 18,
+        "gsd": 30,
```

Comment:
Minor, but maybe move gsd up so that it's basically grouped with the other core fields?